web/src/lib/domain-logic/postOnboardingCheckboxes.ts

```diff
@@ -1,11 +0,0 @@
-type PostOnboardingQuestionId = string;
-type CheckboxId = string;
-
-export const postOnboardingCheckboxes: Record<PostOnboardingQuestionId, CheckboxId[]> = {
-  "construction-renovation": [
-    "construction-renovation-site-drawings",
```

Comment:
dust - what are we doing with the checkbox values for construction? Is a migration needed to remove the values from userData?